src/pudl/analysis/epacamd_eia.py

```diff
@@ -49,15 +48,16 @@ def _get_unique_keys(epacems: pd.DataFrame | dd.DataFrame) -> pd.DataFrame:
     """Get unique unit IDs from CEMS data.
 
     Args:
-        epacems (Union[pd.DataFrame, dd.DataFrame]): epacems dataset from pudl.output.epacems.epacems
+        epacems (Union[pd.DataFrame, dd.DataFrame]): epacems dataset from
+            pudl.output.epacems.epacems
 
     Returns:
         pd.DataFrame: unique keys from the epacems dataset
 
     """
     # The purpose of this function is mostly to resolve the
     # ambiguity between dask and pandas dataframes
-    ids = epacems[["plant_id_eia", "unitid", "unit_id_epa"]].drop_duplicates()
+    ids = epacems[["plant_id_eia", "emissions_unit_id_epa"]].drop_duplicates()
```

Comment:
Could you explain what the new "emissions_unit_id_epa" is in relation to the old "unitid" and "unit_id_epa" identifiers?

Reply:
`emissions_unit_id_epa` == `unitid`
`unit_id` == EPA's unique identifier for `emissions_unit_id_epa` (we removed this field because we can re-calculate it easily)